### ISSUE TITLE
[Feature Request] Multiple Waitlists or Wishlist categories/tags

### ISSUE THREAD

**Issue by tfedor** [IsThereAnyDeal/IsThereAnyDeal-Issues#207]
One thing that has always bugged me about vanilla Steam is that the wishlist is such a one-dimensional thing. I've always wanted a way to either create multiple wishlists or at the very least, categorise accordingly.

I'd love to have the ability through ITAD to create more than just my own waitlist. I'd like one for the games I'd like to buy for my wife. One for games I'd like to buy my son when he's old enough for them. One for my brother, one for my dad, the list goes on. I'd even settle for a tagging/categorising system, where I could assign Myself, Wife, Dad, Brother, Son, etc tags to all waitlist items, and then filter accordingly.

tl;dr - I stumbled upon ITAD while researching Steam/Enhanced Steam APIs to create an application that would enable me to have multiple wishlists, and compare prices between various resellers. Turns out that ITAD is way more polished, feature rich and better in every single way than anything I could ever conceive. Keep up the good work!

[Issue created by Priapist: 2015-06-09]

**Comment by MarkitkaM** [comment]
Possible to create groups for collections and categories for waitlist.
